### ISSUE TITLE
Enhancement: Allow zooming out on sub-1080p monitors.

### ISSUE THREAD

**Issue by GalacticRuler** [endless-sky/endless-sky#814]
some people would love to zoom out a bit on monitors that are not 1980x1080, myself included as I feel on 1600x900 its a tad cramped on screen.

**Comment by Gkramme** [comment]
Would high dpi work for this?

**Comment by GalacticRuler** [comment]
I tried, sadly no.

**Comment by db47h** [comment]
I believe that this is on the [roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap) to 1.0. But what's too big for you? The UI, the main view or both?

For a full zoom out, you can try this:

```
diff --git a/source/Screen.cpp b/source/Screen.cpp
index 5b2e9a3..c10461e 100644
--- a/source/Screen.cpp
+++ b/source/Screen.cpp
@@ -47,7 +47,7 @@ int Screen::Zoom()
 
 void Screen::SetZoom(int percent)
 {
-       ZOOM = max(100, min(200, percent));
+       ZOOM = max(50, min(200, percent));
        WIDTH = RAW_WIDTH * 100 / ZOOM;
        HEIGHT = RAW_HEIGHT * 100 / ZOOM;
 }
```

then manually set the zoom factor to something < 100 in your `~/.local/share/endless-sky/preferences.txt`. Anything below 80 makes the text hardly readable though.

**Comment by lehgogh** [comment]
I believe that ES has too many panels at any given time to not feel cramped on small screens, and we can't directly fix it by removing panels. However, zooming out could be a viable solution.

**Comment by db47h** [comment]
I usually play on a 1080p screen with zoom set to 150% (can hardly read the text without it), so it's like playing with a 720p screen, and yes, it feels a bit cramped, but my biggest gripe is that the main view is too zoomed in: a bactrian takes almost half the screen!

**Comment by tehhowch** [comment]
Additional "UI" zoom increments were added in #3060 
Additional "Space" zoom increments were added in 4207d942d7109f1547052bbcc4e922eef7190c27

You still cannot zoom out of the UI (i.e. make drawn text smaller), but you can make displayed ships smaller or larger. @GalacticRuler 's intent is not explicitly clear, but I think was in reference to the "space" zoom - which would imply this issue can be closed with milestone `v0.9.5 (done)`
